code_schemes/icraf_s01e02.json

```diff
@@ -3,69 +3,373 @@
     "Name": "ICRAF s01e02",
     "Version": "0.0.0.1",
     "Codes": [
-      {
+        {
+        "CodeID": "code-81777bc3",
+        "CodeType": "Normal",
+        "DisplayText": "mathioya arrow roots farmers group",
+        "NumericValue": 0,
+        "StringValue": "mathioya_arrow_roots_farmers_group",
+        "VisibleInCoda": true
+        },
+        {
+        "CodeID": "code-7ef6d529",
+        "CodeType": "Normal",
+        "DisplayText": "gacharu water project",
+        "NumericValue": 1,
+        "StringValue": "gacharu_water_project",
+        "VisibleInCoda": true
+        },
+        {
+        "CodeID": "code-8a12acc3",
+        "CodeType": "Normal",
+        "DisplayText": "green future foundation",
+        "NumericValue": 2,
+        "StringValue": "green_future_foundation",
+        "VisibleInCoda": true
+        },
+        {
+        "CodeID": "code-4d0ba654",
+        "CodeType": "Normal",
+        "DisplayText": "mt kenya ecosystem",
+        "NumericValue": 3,
+        "StringValue": "mt_kenya_ecosystem",
+        "VisibleInCoda": true
+        },
+        {
+        "CodeID": "code-3653f9db",
+        "CodeType": "Normal",
+        "DisplayText": "ktda",
+        "NumericValue": 4,
+        "StringValue": "ktda",
+        "VisibleInCoda": true
+        },
+        {
+        "CodeID": "code-5c8799d6",
+        "CodeType": "Normal",
+        "DisplayText": "rainforest alliance",
+        "NumericValue": 5,
+        "StringValue": "rainforest_alliance",
+        "VisibleInCoda": true
+        },
+        {
+        "CodeID": "code-417b830a",
+        "CodeType": "Normal",
+        "DisplayText": "fairtrade",
+        "NumericValue": 6,
+        "StringValue": "fairtrade",
+        "VisibleInCoda": true
+        },
+        {
+        "CodeID": "code-61f1e435",
+        "CodeType": "Normal",
+        "DisplayText": "kefri",
+        "NumericValue": 7,
+        "StringValue": "kefri",
+        "VisibleInCoda": true
+        },
+        {
+        "CodeID": "code-e594ac70",
+        "CodeType": "Normal",
+        "DisplayText": "gikoe primary school",
+        "NumericValue": 8,
+        "StringValue": "gikoe_primary_school",
+        "VisibleInCoda": true
+        },
+        {
+        "CodeID": "code-86ad27a7",
+        "CodeType": "Normal",
+        "DisplayText": "tist",
+        "NumericValue": 9,
+        "StringValue": "tist",
+        "VisibleInCoda": true
+        },
+        {
+        "CodeID": "code-a78372da",
+        "CodeType": "Normal",
+        "DisplayText": "eurura ecocide warriors shg",
+        "NumericValue": 10,
+        "StringValue": "eurura_ecocide_warriors_shg",
+        "VisibleInCoda": true
+        },
+        {
+        "CodeID": "4b8ecb93",
+        "CodeType": "Normal",
+        "DisplayText": "catholic_diocese_murang'a",
```

Comment:
The DisplayText and StringValue seem to be swapped here

Reply:
Updated thanks